src/main/kotlin/no/nav/familie/ef/sak/brev/dto/FrittståendeBrevRequestDto.kt

```diff
@@ -0,0 +1,9 @@
+package no.nav.familie.ef.sak.brev.dto
+
+import java.time.LocalDate
+
+data class FrittståendeBrevRequestDto(val overskrift: String,
+                                      val avsnitt: List<FrittståendeBrevAvsnitt>,
+                                      val brevdato: LocalDate,
+                                      val ident: String,
```

Comment:
Hvis det er fnr til personen kan vi kanskje bruke `personIdent` her? Vi hadde en diskusjon lenge siden om hva vi skulle bruke for ident/fnr/personIdent og ble då enige om `personIdent`

Reply:
Fikset i a422031